src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/DiagnosticSourceEventSource.cs

```diff
@@ -949,6 +946,12 @@ public static PropertyFetch FetcherForProperty(Type? type, string propertyName)
                                 Logger.Message($"Property {propertyName} not found on {type}");
                                 return new PropertyFetch(type);
                             }
+                            // Delegate creation below is incompatible with static properties.
```

Comment:
Is this an existing bug in the code? This doesn't seem related to the linker.

Reply:
I logged https://github.com/dotnet/runtime/issues/38406.